pkg/imagesystem/registry.go

```diff
@@ -189,7 +189,7 @@ func isNotInternalRepo(prefix, image string) error {
 	}
 
 	if prefix == "" {
-		return nil
+		prefix = fmt.Sprintf("%s.%s", system.RegistryName, system.ImagesNamespace)
```

Comment:
I updated this check so that it works with the default internal registry (`registry.acorn-image-system`) if nothing else is specified.